source/features/reactions-avatars.css

```diff
@@ -21,6 +21,12 @@ button.reaction-summary-item { /* `button` excludes the "Add reaction" icon */
 	--background: #111d2f;
 }
 
+@media (prefers-color-scheme: 'dark') {
+	[data-color-mode='auto'] .reaction-summary-item.user-has-reacted {
+		--background: #111d2f;
```

Comment:
It looks like `--background` can be dropped everywhere and `box-shadow` should use `var(--color-bg-info)` directly

Reply:
@fregante There is a use of `--background` for `.reaction-summary-item a img` on line 51 ( 57 ). I think it is unnecessary. shall I take that out as well?

Reply:
No, there's a reason why it's there: https://github.com/sindresorhus/refined-github/pull/645